.github/workflows/release-layer-java.yml

```diff
@@ -8,11 +8,20 @@ on:
 
 permissions:
   id-token: write
-  contents: read
+  contents: write
 
 jobs:
+  create-release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Create Release
+        run: gh release create ${{ github.ref_name }} --draft
```

Comment:
`github.ref_name` will be tag name (and so version) in practice, right? So, release name will include version number?

Reply:
If so, can we use specify a custom title for the release by using tag (version number) over `github.ref_name`?

Reply:
So I updated the latest version to set the title to match what is currently used for release titles.

<img width="1091" alt="image" src="https://github.com/user-attachments/assets/55265583-8208-426e-b467-62f33506b88e">

in this case the tag/version number is used. This matches what I see in the current releases here: 
![image](https://github.com/user-attachments/assets/06940228-5d52-486c-b961-54bab292c195)

I am going to mark this comment as resolved. If I misunderstood, please feel free to reopen.